_data/packages.yml

```diff
@@ -1,570 +1,42 @@
-        - 2017
-    - name: BigchainDB
-      id: bigchain
-      url: http://www.bigchaindb.com/
-      text: |-
-        <a href="http://www.bigchaindb.com/" target="_blank">BidchainDB</a> allows developers and enterprise to deploy blockchain proof-of-concepts,
-         platforms and applications with a scalable blockchain database. BigchainDB supports a wide range of industries and use cases from identity 
-         and intellectual property to supply chains, energy, IoT and financial ecosystems. With high throughput, sub-second latency and powerful functionality 
-        to automate business processes, BigchainDB looks, acts and feels like a database but has the core blockchain characteristics that enterprises want.
-      years:
-        - 2017
-    - name: Nextjournal
-      id: nextjournal
-      url: https://nextjournal.com
-      text: |-
-        <a href="https://nextjournal.com" target="_blank">Nextjournal</a> Nextjournal makes research explorable - uniting prose, code, data and visualizations
-         in beautiful, ready-to-run, living documents. We love open source and are excited to support RGSoC 2017 and see what you build!
-      years:
-        - 2017
-    - name: Manifold
-      id: manifold
-      url: https://www.manifold.co/
-      text: |-
-        <a href="https://www.manifold.co/" target="_blank">Manifold</a> is the best way to add cloud services to your application on any cloud, made by developers, 
-        with love. We are honored to be able to support diversity and empowerment in Open Source through such a wonderful cause. Excited to see what y'all build! 🤗
-      years:
-        - 2017
+#- name: Partners
+#  title: Partner Companies
+#  sponsors:
+#    - name: Travis CI
+#      id: travis
+#      url: http://travis-ci.com
+#      partner: true
+#      text: |-
+#        <a href="https://travis-ci.com" target="_blank">Travis</a> is an Open Source
+#        company, born and raised in beautiful Berlin. Travis CI is their hosted
+#        continuous integration service for the open source community. It is integrated
+#        with GitHub and offers first class support for a ton of programming languages,
+#        like Python, Go, PHP, Java, C++ and of course Ruby. They also launched <a
+#        href="http://foundation.travis-ci.org">Travis Foundation</a>, a non-profit
+#        organisation which aims to make Open Source an even better place. One of their
+#        goals is to make Rails Girls Summer of Code a recurring tradition.
+#      years:
+#        - 2013
+#        - 2014
+#        - 2015
+#        - 2016
+#        - 2017
+#        - 2018
+#- name: Platinum
+#  title: Platinum Sponsors
+#  sponsors:
+#- name: Gold
+#  title: Gold Sponsors
+#  sponsors:
+#- name: Silver
+#  title: Silver Sponsors
+#  sponsors:
+#    - name: Discourse
+#      id: discourse
+#      url: https://www.discourse.org/
+#      #text: |- 
+#      years:
+#        - 2018
+#- name: Bronze
+#  title: Bronze Sponsors
+#  sponsors:
```

Comment:
we could already add Loco2 (the latest sponsor) to this :)

Reply:
One of the reasons I commented all of this is that I don't have the logos from the new sponsors and that raises an annoying error. How do you handle this? Doss the logo and text comes from the sponsors or should someone from RGSoC pick this from the internet?
Also, how do you want to handle the Travis sponsorship? Do we keep the same text or change it for Travis Foundation? In that case we'll need to update the Travis contribution in the donation page, as we talked.

Reply:
One of the reasons I commented all of this is that I don't have the logos from the new sponsors and that raises an annoying error. How do you handle this? Doss the logo and text comes from the sponsors or should someone from RGSoC pick this from the internet?
Also, how do you want to handle the Travis sponsorship? Do we keep the same text or change it for Travis Foundation? In that case we'll need to update the Travis contribution in the donation page, as we talked.